ext/settings.html

```diff
@@ -2487,6 +2488,27 @@ <h5>or click here to upload</h5>
     </div>
 </div></div>
 
+<div id="dictionary-confirm-update-modal" class="modal" tabindex="-1" role="dialog" hidden><div class="modal-content modal-content-small">
+    <div class="modal-header"><div class="modal-title">Confirm Dictionary Update</div></div>
+    <div class="modal-body">
+        <p>Are you sure you want to update the dictionary:</p>
+        <p><strong id="dictionary-confirm-update-name"></strong>?</p>
+        <section>
+            Updating a dictionary involves:
+            <ul>
+                <li>Deleting the installed version</li>
+                <li>Downloading the latest version </li>
+                <li>Importing the latest version</li>
+            </ul>
+            <p class="warning-text">Especially for large dictionaries, this process can take a while, and downloading will use your network.</p>
```

Comment:
Does closing the settings page during dictionary import interrupt the dictionary import process? I wonder if we should put a warning to tell people not to close their browsers?

Reply:
There's an alert 
![image](https://github.com/themoeway/yomitan/assets/24891609/070f6be4-1af1-4d38-9c46-4d973fb61401)